jscomp/main/bsb_main.ml

```diff
@@ -181,7 +175,16 @@ let install_target config_opt =
 let () =
   try begin 
     match Sys.argv with 
+    | [| _; "-backend"; _ |]
```

Comment:
is there any way to make this guarded on native?

Reply:
Of course, it'll look ugly with the two `#if` in a row, but because of the `->` syntax I cannot do it in one `#if` :(